io.github.lxi-tools.yaml

```diff
@@ -0,0 +1,133 @@
+---
+app-id: io.github.lxi-tools.lxi-gui
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: lxi-gui
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=home"
+modules:
+  - name: lua
+    sources:
+      - type: archive
+        url: https://www.lua.org/ftp/lua-5.3.6.tar.gz
+        sha256: fc5fd69bb8736323f026672b1b7235da613d7177e72558893a0bdcd320466d60
+      - type: patch
+        path: lua-makefile-is-broken.patch
+      - type: file
+        path: lua-5.3.pc
+    buildsystem: simple
+    build-commands:
+      - make linux INSTALL_TOP=${FLATPAK_DEST} MYCFLAGS=-fPIC V=5.3 R=5.3.6 install
+      - mkdir -p ${FLATPAK_DEST}/lib/pkgconfig
+      - cp lua-5.3.pc ${FLATPAK_DEST}/lib/pkgconfig
+
+  - name: libtirpc
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.2.tar.bz2
+        sha256: e24eb88b8ce7db3b7ca6eb80115dd1284abc5ec32a8deccfed2224fc2532b9fd
+    buildsystem: autotools
+
+  - name: readline
+    sources:
+      - type: archive
+        url: https://mirrors.dotsrc.org/gnu/readline/readline-8.1.tar.gz
+        sha256: f8ceb4ee131e3232226a17f51b164afc46cd0b9e6cef344be87c65962cb82b02
+    buildsystem: autotools
+
+  - name: rpcsvc-proto
+    sources:
+      - type: archive
+        url: https://github.com/thkukuk/rpcsvc-proto/releases/download/v1.4.2/rpcsvc-proto-1.4.2.tar.xz
+        sha256: 678851b9f7ddf4410d2859c12016b65a6dd1a0728d478f18aeb54d165352f17c 
+    buildsystem: autotools
+
+  - name: libevent
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+    buildsystem: autotools
+
+  - name: avahi
+    sources:
+      - type: archive
+        url: http://avahi.org/download/avahi-0.8.tar.gz
+        sha256: 060309d7a333d38d951bc27598c677af1796934dbd98e1024e7ad8de798fedda
+    buildsystem: autotools
+    config-opts:
+      - "--with-distro=lfs"
+      - "--disable-python"
+      - "--disable-qt4"
+      - "--disable-qt5"
+      - "--disable-gtk"
+      - "--disable-libdaemon"
+      - "--disable-mono"
+
+  - name: liblxi
+    sources:
+      - type: archive
+        url: https://github.com/lxi-tools/liblxi/releases/download/v1.15/liblxi-1.15.tar.xz
+        sha256: c334ee3c4cc668821f356181afa1cd33e7113aef5464b187ac5b842059abb8b4
+    buildsystem: meson
+
+  - name: pango
```

Comment:
It's already in the runtime.

Reply:
Sorry, can't remove that, I need most recent GTK4 which requires newer pango that what is in your runtime.